docs/sync_code.md

```diff
@@ -0,0 +1,34 @@
+# Basic workflow for Operator
+
+1. Create a branch:
+Start by creating a branch from "main" branch.
+
+2. Ensure local tests pass:
+Run unit and e2e tests, and ensure they pass before submitting a pull request (PR) targeting "main" branch.
+If the PR is associated with any Jira ticket, include the ticket link in the "Description" section of the PR.
+
+
+3. Review and merge PR:
+Once the PR has passed github checks and is reviewed and approved, the PR author should merge it into "main" branch.
+
+4. Create a downstream sync PR:
+The PR author should then create another PR targeting "rhoai" branch. Preferably, the PR title should include the prefix [sync], and the "rhoai" label should be added to help reviewers quickly acknowledge it.
+If the PR is associated with any Jira ticket, include the ticket link in the "Description" section of the PR.
+
+5. Merge sync PR:
+After the sync PR has passed github checks and is reviewed and approved, the PR author should merge it into "rhoai" branch.
+
+6. Automated sync:
+Automation will sync changes to the RHDS rhods-operator on a daily basis.
+
+# PR labels
```

Comment:
Are we allowed to set labels?

Reply:
I was hoping that the /label command could be used for that, but it seems that it's a static list of labels specific to openshift that are allowed for that. Maybe  something could be added to the config in openshift/release for it, not sure :thinking:

Reply:
i think manitainers should have permission to set label.
at least i used labels a lot both for my PR and the PR i was asked to review or triage.


also i do not know how it will be with these prow bot, when we move to konflux, do we still need prow, will /cmd still work etc.
if we do not want to depend on an external system, e.g prow, native feature of GH like  create/add labels should always work, (i hope)